packages/device-utils/src/modelUtils.ts

```diff
@@ -1,3 +1,44 @@
 import { PartialDevice } from './types';
+import { getFirmwareMajorVersion } from './firmwareUtils';
 
-export const getDeviceModel = (device?: PartialDevice) => device?.features?.model || '';
+export enum DeviceModel {
+    T1 = '1',
+    TT = 'T',
+    TR = 'R',
+    UNKNOWN = '',
```

Comment:
is it useful?

Reply:
I would say so 😄

Reply:
Removed

Reply:
Yes, device is not always defined and if device is not defined, then `getDeviceModel` returns `DeviceModel.Unknown`